processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -63,6 +64,10 @@ type WatchClient struct {
 // format: [deployment-name]-[Random-String-For-ReplicaSet]-[Random-String-For-Pod]
 var dRegex = regexp.MustCompile(`^(.*)-[0-9a-zA-Z]*-[0-9a-zA-Z]*$`)
 
+// Extract CronJob name from the Job name. Job name is created using
+// format: [cronjob-name]-[time-hash-int]
+var cronJobRegex = regexp.MustCompile(`^(.*)-[0-9]*$`)
```

Comment:
```suggestion
var cronJobRegex = regexp.MustCompile(`^(.*)-[0-9]+$`)
```

Since we don't want to match a manually created job like `mycooljob-` :)